### ISSUE TITLE
K1 OS not installing OctoEverywhere venv

### ISSUE THREAD

**Issue by EchotreePrints** [QuinnDamerell/OctoPrint-OctoEverywhere#82]
Greetings, Programs!

I want to start by thanking you for making this script, and having it run on the Creality K1 series printers! Also, that I gave virtually no knowledge or experience with linux, outside of a few raspberry pi projects and the K1.

I've run into an issue where the installer script either can't locate, or can't create the venv for the octoeverywhere script to run in, so the script can install, the components but not run. I'm not sure what else to add here to get you the information you need, so let me know if you need more info. I've tried following along on some of the other issues K1 users opened here, but they either seemed unrelated or the fixes didn't work.

![wtf](https://github.com/user-attachments/assets/bc9ee142-8142-4170-b1fd-49eeac190c04)

**Comment by QuinnDamerell** [comment]
Sorry for the delay! Were you able to figure this out? Were you installing using the creality installer script?

**Comment by EchotreePrints** [comment]
> Sorry for the delay! Were you able to figure this out? Were you installing using the creality installer script?

I have not been able to figure this out, sadly. Like I said, I know very little about linux. I am using a Creality K1 with the Helper script, but even just trying to install the plugin by itself, I continue to run into this error.

**Comment by EchotreePrints** [comment]
I'm going to preface this again by saying I have no experience with linux, and that's probably why I ran into this issue, BUT I've found a fix. It's The same advice @QuinnDamerell gave to another user on another issue, but I think I interpreted it wrong the first time reading it.

The fix:
`cd ~

rm -rf /usr/data/octoeverywhere-env /usr/data/octoeverywhere

cd /usr/data

git clone https://github.com/QuinnDamerell/OctoPrint-OctoEverywhere.git octoeverywhere

cd octoeverywhere

sh install.sh`

This removes the broken venv created by the Helper-Script and the octoeverywhere folder installed by the helper script. Once this is done, we clone the official repository back to where it's supposed to be, and run the install script. For whatever reason, I never thought to remove the octoeverywhere-env directory, even after trying to install from the official repository.

This fix is courtesy of u/predtech on reddit, who received it from Destinal in the Devil Design discord server. 

Thank you for your Time, and again for making a wonderful service!
